src/czar/CzarRegistry.cc

```diff
@@ -80,13 +80,15 @@ void CzarRegistry::_registryUpdateLoop() {
     string const url = "http://" + _czarConfig->replicationRegistryHost() + ":" +
                        to_string(_czarConfig->replicationRegistryPort()) + "/czar";
     vector<string> const headers = {"Content-Type: application/json"};
+    LOGS(_log, LOG_LVL_INFO, "CzarRegistry::_registryUpdateLoop() getting FQDN");
```

Comment:
Do you want to see this message in the elog stream? You could downgrade it to be at most `DEBUG` or maybe `TRACE`. A better option is not to pollute the code or the log stream with this.

Reply:
If the FQDN can't be obtained then an exception is thrown (by the called method). In this case, it doesn't matter from what context you're calling this. No one is going to investigate this. What would matter here is that there is a problem with the system/networking infrastructure.